index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style.css">
+    <title>Document</title>
```

Comment:
@nvxquynhsgt 
- Thông thường thẻ `<link>` nằm dưới thẻ `<title>`, `<meta>` nằm trên `<title>`
- Sửa title thành `Poll App` 👍 
```suggestion
    <title>Poll App</title>
    <link rel="stylesheet" href="/style.css">
```